round-2

AliIbadullayev opened the pull request:
Changed way of rounding to fixed length a double

ryukzak commented:
Thank you! Can you extract this function to misc module and use in all such places?

AliIbadullayev commented:
* Rewrote test in modern way  according to #15
* Added 'to-fixed' func in misc module and replaced in all places where used  
* Removed unused functions and unused modules from headers 
* Added some other function to misc module: `quiz-answ`

ryukzak commented:
Sorry, but can you split it into several PRs by their purpose? Contact me by telegram.

ryukzak commented on src/course_bot/misc.clj:
```diff
@@ -36,3 +36,5 @@
 (defn normalize-time [dt]
   (str-time (read-time dt)))
+
+(defn to-fixed [num len] (double (/ (Math/round (* (Math/pow 10 len) num)) (Math/pow 10 len))))
```
Please, rename it to round-2

AliIbadullayev replied:
done

ryukzak reviewed (changes requested):
Also, can you squash all you commits in one.
